manual/logging/README.md

```diff
@@ -113,78 +113,73 @@ and provide hints about what's going wrong.
 
 ### Logging query latencies
 
-The `QueryLogger` provides clients with the ability to log queries
-executed by the driver, and especially, it allows client to track slow
+The `EnhancedQueryLogger` class provides clients with the ability to log queries
+executed by the driver, and especially, it allows clients to track slow
 queries, i.e. queries that take longer to complete than a configured
 threshold in milliseconds.
 
-To turn on this feature, you first need to instantiate and register a `QueryLogger` instance:
+To turn on this feature, you first need to instantiate and register an `EnhancedQueryLogger` instance:
 
 ```java
 Cluster cluster = ...
-QueryLogger queryLogger = QueryLogger.builder()
+EnhancedQueryLogger queryLogger = EnhancedQueryLogger.builder()
     .withConstantThreshold(...)
-    .withMaxQueryStringLength(...)
 .build();
 cluster.register(queryLogger);
 ```
 
-Note that `QueryLogger` instances are thread-safe and can be shared cluster-wide. 
-Besides, you can adjust several parameters such as the maximum query string length to be printed, 
-the maximum number of parameters to print, etc. Refer to the 
-`QueryLogger` [API docs][query_logger] for more information.
+Note that `EnhancedQueryLogger` instances are thread-safe and can be shared cluster-wide. 
+Besides, you can adjust several parameters. Refer to the 
+`EnhancedQueryLogger` [API docs][query_logger] for more information.
 
-Secondly, you need to adjust your logging framework to accept log messages from the `QueryLogger`. The `QueryLogger`
+Secondly, you need to adjust your logging framework to accept log messages from the `EnhancedQueryLogger`. The `EnhancedQueryLogger`
 uses 3 different loggers:
 
-* `com.datastax.driver.core.QueryLogger.NORMAL` : Used to log normal queries, i.e., queries that completed successfully within a configurable threshold in milliseconds.
-* `com.datastax.driver.core.QueryLogger.SLOW` : Used to log slow queries, i.e., queries that completed successfully but that took longer than a configurable threshold in milliseconds to complete.
-* `com.datastax.driver.core.QueryLogger.ERROR`: Used to log unsuccessful queries, i.e., queries that did not complete normally and threw an exception. Note this this logger will also print the full stack trace of the reported exception.
+* `com.datastax.driver.core.EnhancedQueryLogger.NORMAL` : Used to log normal queries, i.e., queries that completed successfully within a configurable threshold in milliseconds.
```

Comment:
What do you think about reusing `QueryLogger` for the logger class?  I imagine that in next major release, we will simply get rid of the existing `QueryLogger` and rename `EnhancedQueryLogger` to `QueryLogger`.   If that is the case, why not reuse `QueryLogger` as the class that the logger uses?   I don't expect users to simultaneously want to configure both.   I could be otherthinking it :D